### DISCUSSION TITLE
Any discussion on game dev is welcome 👍 !

### DISCUSSION THREAD

**Discussion by MarkfuGod** [MarkfuGod/roguelikeDev#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fidoxas** [comment]
Hi, I'm Fido, a young game developer. I've created a mobile game and am currently in the process of further development. I'm here to learn more about promoting games and to connect with ambitious individuals who share a passion for creating something amazing together. In the future, I aspire to develop a computer game with a compelling plot, visually stunning world, and cool mechanics. Let's collaborate and make something great!
